.circleci/config.yml

```diff
@@ -34,6 +34,10 @@ commands:
           - linux
           - windows
         type: enum
+      lts:
```

Comment:
We're using a boolean parameter here because it makes the internals of this command much less complicated that if we were to use an enum.